src/main/java/sirius/biz/codelists/CustomLookupTable.java

```diff
@@ -118,4 +119,14 @@ public int count() {
         return Stream.concat(customTable.performQuery(lang, lookupPath, lookupValue),
                              baseTable.performQuery(lang, lookupPath, lookupValue));
     }
+
+    @Override
+    public String getTitle() {
+        return extension.getId().equals(super.getTitle()) ? baseTable.getTitle() : super.getTitle();
```

Comment:
shoudln't re rather directly ask NLS.getIfExists or the like here? This seem sort of hacky..

Reply:
Why should we treat the tables differently? I think we should use the same method to find texts for both

Reply:
ah wait, you're right... This would also return the base table's name instead of the name of the custom table. That is an issue... will change to the NLS.getIfExists